tests/utils/test_validation.py

```diff
@@ -44,9 +44,10 @@ def test_validate_tag_name():
 
 
 def test_validate_run_id():
-    for good_id in ["a" * 32, "f0" * 16, "abcdef0123456789" * 2]:
+    for good_id in ["a" * 32, "f0" * 16, "abcdef0123456789" * 2, "a" * 33, "a" * 31,
```

Comment:
Could be nice to test a length-255 string (like "a" * 255) and a string with non-hex alphanumeric characters (like "abcdefghijklmnopqrstuvwxyz") just to be super safe :)